src/App/App.js

```diff
@@ -27,9 +27,9 @@ class App extends Component {
   this.getFavorites();
  }
 
-//   componentDidUpdate = (prevState) => {
- 
-//  }
+  checkFavorites = (id) => {
```

Comment:
Love that we can use this to return truthy or falsey to help us to set the state of our movieDisplay.